## Graph Machine Learning Resources for Drug Discovery

Explore a curated collection of databases and resources tailored for graph machine learning applications in drug discovery. These resources offer comprehensive datasets, tools, and benchmarks to propel your research in understanding complex biological systems and advancing therapeutic innovations.

**Open Graph Benchmark (OGB)**
   - **Link**: [OGB](https://ogb.stanford.edu/)
   - **Description**: A collection of benchmark datasets, data loaders, and evaluators for graph machine learning. It includes specific datasets for drug discovery tasks like molecular property prediction and protein-protein interaction networks. The OGB datasets are particularly designed to standardize and facilitate graph research, providing diverse, large-scale, and challenging datasets that are useful for developing and benchmarking GNN models in drug discovery.

**Hetionet**
   - **Link**: [Hetionet](https://neo4j.het.io/browser/)
   - **Description**: A hetnet (heterogeneous network) of relationships between different types of entities, including diseases, genes, compounds, and pathways. It's a valuable resource for studying drug repurposing and drug-disease associations using GNNs.

**Graph4Med**
   - **Link**: [Graph4Med](http://graph4med.cs.uni-frankfurt.de), [Github](https://github.com/jeschaef/Graph4Med)
   - **Description**: t Graph4Med, a web application that relies on a Neo4J graph database obtained by transforming a traditional relational database. Graph4Med provides a straightforward visualization and analysis of a selected patient cohort. 

**CROssBAR:** Comprehensive Resource of Biomedical Relations with Deep Learning Applications and Knowledge Graph Representations
   - **Link**: [CROssBAR](https://crossbar.kansil.org/)

   - **Description**:
   CROssBAR is a comprehensive system that integrates large-scale biomedical data from various resources and store it in a new NoSQL database, enrich this data with deep learning based prediction of relations between numerous biomedical entities, rigorously analyse the enriched data to obtain biologically meaningful modules and display them to the user via easy to interpret, interactive and heterogenous knowledge graphs within an open access, user-friendly and online web-service.
   
**ZINC Database**
   - **Link**: [ZINC Database](https://zinc.docking.org/)
   - **Description**: A free database of commercially available compounds for virtual screening. It contains over 35 million compounds and is widely used for drug discovery projects, particularly in tasks like molecular generation and optimization.

**PubChem**
   - **Link**: [PubChem](https://pubchem.ncbi.nlm.nih.gov/)
   - **Description**: A public database of chemical molecules and their activities against biological assays. PubChem is used extensively for structure-activity relationship studies and molecular property prediction.

**ChEMBL**
   - **Link**: [ChEMBL](https://www.ebi.ac.uk/chembl/)
   - **Description**: A manually curated database of bioactive molecules with drug-like properties. It provides information on compound bioactivity, targets, and pharmacological data, useful for GNN models that predict drug-target interactions and drug efficacy.

**PDBbind**
   - **Link**: [PDBbind](http://www.pdbbind.org.cn/)
   - **Description**: A comprehensive collection of the binding affinities of proteins and ligands. It's a crucial resource for developing models that predict protein-ligand interactions.

**DrugBank**
   - **Link**: [DrugBank](https://www.drugbank.ca/)
   - **Description**: A unique bioinformatics and cheminformatics resource combining detailed drug data with comprehensive drug target information. It's used for drug-target interaction studies.

**MoleculeNet**
   - **Link**: [MoleculeNet](http://moleculenet.ai/)
   - **Description**: A benchmark dataset for molecular machine learning. It covers a range of molecular properties, including quantum mechanics, physical chemistry, biophysics, and physiology.

**BindingDB**
   - **Link**: [BindingDB](https://www.bindingdb.org/bind/index.jsp)
   - **Description**: A public, web-accessible database of measured binding affinities, focusing on the interactions of proteins considered to be drug targets with ligands.

**Tox21**
   - **Link**: [Tox21](https://tripod.nih.gov/tox21/challenge/)
   - **Description**: A dataset from the Toxicology in the 21st Century initiative, containing data on the toxicity of compounds, which is essential for ADMET studies.

**Human Metabolome Database (HMDB)**
   - **Link**: [HMDB](http://www.hmdb.ca/)
   - **Description**: Provides comprehensive information on human metabolites and their roles in metabolism. Useful for understanding drug metabolism and pharmacokinetics in GNN models.

**Genomics of Drug Sensitivity in Cancer (GDSC)**
   - **Link**: [GDSC](https://www.cancerrxgene.org/)
   - **Description**: Offers a rich dataset of drug responses in cancer cell lines, including genetic, molecular, and pharmacological data. Ideal for GNNs focusing on cancer therapeutics and personalized medicine.

**Therapeutic Target Database (TTD)**
   - **Link**: [TTD](http://db.idrblab.net/ttd/)
   - **Description**: Provides information about known and explored therapeutic protein and nucleic acid targets, the targeted disease, pathway information, and the corresponding drugs directed at each of these targets.

**SureChEMBL**
   - **Link**: [SureChEMBL](https://www.surechembl.org/)
   - **Description**: Contains chemical information extracted from the patent literature, useful for novel chemical entity discovery and GNN models focusing on intellectual property in drug design.

**Protein Data Bank (PDB)**
   - **Link**: [PDB](https://www.rcsb.org/)
   - **Description**: A global repository of information about the 3D structures of large biological molecules, including proteins and nucleic acids. Essential for protein structure prediction and ligand docking studies.

**Connectivity Map (CMap)**
   - **Link**: [CMap](https://clue.io/)
   - **Description**: Provides gene expression profiles in response to various chemical and genetic perturbations, aiding in understanding drug mechanisms of action and potential off-target effects.

 **ChemSpider**
   - **Link**: [ChemSpider](http://www.chemspider.com/)
   - **Description**: A free chemical structure database providing fast access to over 100 million structures, properties, and associated information. Useful for sourcing chemical compounds and property data.

**STITCH**
   - **Link**: [STITCH](http://stitch.embl.de/)
   - **Description**: Integrates information about interactions between chemicals and proteins, including metabolic and signaling pathways, chemicals, and drugs. It's valuable for studying drug-protein interaction networks.

**LINCS L1000**
   - **Link**: [LINCS L1000](http://www.lincsproject.org/LINCS/)
   - **Description**: Provides gene expression signatures in response to a variety of drug and genetic perturbations, facilitating the understanding of compound mechanisms of action.
